docs/application/vstools/Tizen/native.md

```diff
@@ -52,3 +52,13 @@ To create a Tizen Native project:
    ![Add break point](media/native_debug_application.PNG)
 
 3. Start the debugging session by selecting **Debug &gt; Start Debugging** in the menu bar, or pressing **F5**, or by clicking the **Debug** button in the menu bar.
+
```

Comment:
The whole list is not highlighted. Only the first point is highlighted out of the three.